### PR TITLE
Hcap 240 moh request access

### PR DESCRIPTION
**Pull request by Zenen-freshworks**

### CODE REVIEW

File: client/src/pages/private/UserView.js [hunk 1]
```diff
@@ -220,6 +223,39 @@ export default () => {
                   }}
                 />
               </Box>}
+            {(values.role === 'employer') && <Box mt={3}>
```
Comment (fwkendall) [inline client/src/pages/private/UserView.js:226]: Isn't this condition already handled on line 204? I'm confused why we check for employer twice

File: client/src/pages/private/ParticipantView.js [hunk 2]
```diff
@@ -335,7 +335,8 @@ export default () => {
         setLoadingUser(false);
         setSites(sites);
         setRoles(roles);
-        
+        // a Boolean filter removes all undefined values
+        setLocations(roles.map((loc) => locationRoles[loc]).filter(Boolean));
```
Comment (fwkendall) [inline client/src/pages/private/ParticipantView.js:339]: Curious why this was added? I thought the tests might need it but that doesn't seem to be the case :)

**Comment by fwkendall** [comment]
Looks good! (And tests look great) - just a couple of questions from me :)

File: client/src/pages/private/UserView.js (hunk 1 above)
Comment (Zenen-freshworks) [reply client/src/pages/private/UserView.js:226]: Good catch! This was some redundant code that I was going to use as a template and then forgot to remove it.

File: client/src/pages/private/ParticipantView.js (hunk 2 above)
Comment (Zenen-freshworks) [reply client/src/pages/private/ParticipantView.js:339]: More redundant code! I implemented a better version of this setter a few lines down.

**Review by fwkendall — APPROVED** [review]

**Review by mauriciofreshw — APPROVED** [review]

**Pull request merged**